server/Controllers/PointsController.cs

```diff
@@ -109,6 +115,10 @@ public IActionResult PostOne(Guid id, [FromBody]Point point)
             {
                 return new BadRequestObjectResult(ex.ToString());
             }
+            if (points.ContainsKey(point.Id))
+            {
+                _logger.LogWarning($"Overwriting the point with id {point.Id}");
```

Comment:
nit: is this the only place we need logger; what do you think to use it for some exceptions (if not for all exceptions)?